docs/changes.rst

```diff
@@ -36,70 +36,68 @@ details of the changes themselves and the reasons for the changes.
 
 *Summary of incompatible changes:*
 
+
+The details, alternatives, and reasons for these incompatible changes is shown
+below this list.
+
 * Removed Python 2.6 support.
 
 * Migrated pywbem to use the 'requests' Python package for HTTP/HTTPS pywbem
-  client to WBEM server communication. This removed support for UNIX sockets
-  based connections to local WBEM servers as well as the non-standard 'OWLocal'
-  and 'Local' authentication schemes, and changed some of the behavior of
-  HTTPS-related init parameters of `WBEMConnection`.
+  client to WBEM server communication.
 
-* Removed the deprecated `verify_callback` init parameter of `WBEMConnection`.
+* Removed the following deprecated functionality:
 
-* Removed the `**extra` keyword arguments from `WBEMConnection` operation
-  methods.
+  - `WBEMConnection` `verify_callback` init parameter.
 
-* Removed the deprecated ordering for `NocaseDict`, `CIMInstanceName`,
-  `CIMInstance` and `CIMClass` objects.
+  - `WBEMConnection` `**extra` keyword arguments from operation methods.
 
-* Removed the deprecated ability to set the following properties of class
-  `WBEMConnection`: `url`, `creds`, `x509`, `ca-certs`, `no_verification`,
-  and `timeout`.
+  - Ordering for `NocaseDict`, `CIMInstanceName`, `CIMInstance` and `CIMClass`
+    objects.
 
-* Removed the deprecated methods `method_call()` and imethod_call()` and the
-  deprecated property `operation_recorder` from class `WBEMConnection`.
+  - `WBEMConnection` properties: `url`, `creds`, `x509`, `ca-certs`,
+    `no_verification`, and `timeout` setter methods. They are now read-only
 
-* Removed the deprecated property `property_list` and the same-named init
-  parameter from class `CIMInstance`.
+  - `WBEMConnection` `method_call()` and imethod_call()` methods.
 
-* Removed the deprecated ability to support a value of `None` for
-  `pywbem.tocimxml()`.
+  - `WBEMConnection` `operation_recorder` property.
 
-* Removed the deprecated `indent` parameter of `CIMInstance.tomof()`.
+  - `CIMInstance` property `property_list` and the same-named init parameter.
 
-* Removed the deprecated internal function `pywbem.byname()`.
+  - `pywbem.tocimxml()` support for value of `None`.
+
+  - `CIMInstance.tomof()`  `indent` parameter.
+
+  - `pywbem.byname()` internal function.
 
-* Removed the deprecated function `pywbem.tocimobj()`.
+  - `pywbem.tocimobj()` function.
 
-* Removed the deprecated `wbemcli` command.
+  - `wbemcli` command.
 
 * Made the `MOFWBEMConnection` class (support for the MOF compiler) internal.
 
-* Exception changes:
+* Changed some exceptions:
```

Comment:
In the spirit of "one short sentence", most of the sub-items for exceptions can be shortened.
For example: 
```
* Changed exception behavior:
  - MOF compilation methods of `MOFCompiler` and `FakedWBEMConnection`.
  - Some methods of `ValueMapping`.
  - Some methods of `WBEMServer`.
  - Added a new exception `pywbem.HeaderParseError` derived from
    `pywbem.ParseError`.
```

Reply:
On the fourth exception item, the methods should be mentioned that may raise the new exception.

Reply:
DONE.